### PR TITLE
fix(DB/Event): Scourge Invasion creature_template, quest mail, loot

### PR DESCRIPTION
**Pull request by sogladev**
## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes issues reported by @Grimfeather https://github.com/azerothcore/azerothcore-wotlk/pull/22286#issuecomment-3075064492

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [ ] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [ ] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [ ] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by heyitsbench — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1752706389703869030.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+--
+-- Bone Witch
+UPDATE `creature_template` SET `minlevel` = 71, `maxlevel` = 71 WHERE (`entry` = 16380);
+
+UPDATE `quest_template_addon` SET `RewardMailDelay`=2*24*3600 WHERE `ID` IN (9295, 9299, 9300, 9301, 9302, 9304);
+
+-- Delete rare loot (lvl 60) and add epic loot (copied from Bone Witch - 16380)
+DELETE FROM `creature_loot_template` WHERE (`Entry` = 16143) AND `Item` IN (23085, 23087, 22484, 23090, 23091, 23092, 23093);
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
```
Comment (heyitsbench) [inline data/sql/updates/pending_db_world/rev_1752706389703869030.sql:9]: I know they came from another creature, but I really am not a fan of these drop chances.
